src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain
+
+class Car(val name: String, var position: Int) {
```

Comment:
아래 `constructor(name: String) : this(name, 0)` 부분은
생성자 부분에서 합쳐서 표현할 수 있어요! 
```suggestion
class Car(val name: String, var position: Int = 0) {
```

Reply:
와 멋져요